runtime_config_linux.go

```diff
@@ -198,6 +198,12 @@ type Network struct {
 	Priorities []InterfacePriority `json:"priorities"`
 }
 
+// IntelRdt for Linux cgroup 'intel_rdt' resource management
+type IntelRdt struct {
+	// L3 cache capacity bitmask (CBM) for container
+	L3Cbm *string `json:"l3Cbm,omitempty"`
```

Comment:
>wking added a note 18 hours ago

>The counter-argument here is that the target audience for the
>config-file isn't end users [1,2,3], which can always insert some
>tooling in between themselves and the JSON config.  For example, it
>wouldn't be too bad to write a parser that converts a JSON-extension
>form like:

@wking It makes sense.If JSON or similar config-file is not always UI-like thing,
integer is better than string even for 'bitmask' in the case.

Actually, I tried to use type 'uint64' instead of 'string' in the v1 patch of 
https://github.com/opencontainers/runc/pull/447, the code looks better, and the JSON config file looks like:

```json
"intel_rdt": {
    "l3Cbm": 4080
}
```
@others, please tell me if you have different opinions.
Otherwise, I will change the type of *IntelRdt.L3Cbm* from *string to *uint64 later.